presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -399,11 +409,9 @@ public JoinNode join(JoinNode.Type joinType, PlanNode left, PlanNode right, Join
                         .addAll(left.getOutputSymbols())
                         .addAll(right.getOutputSymbols())
                         .build(),
+                filter,
                 Optional.empty(),
-                Optional.empty(),
-                Optional.empty(),
-                Optional.empty()
-        );
+                Optional.empty());
```

Comment:
It is related - I've added new overloads of the method. The normal thing to do in such case is to delegate to the new overloads if possible.